### PR TITLE
feat(em): Admin and Poll Worker Smartcard encoding.

### PR DESCRIPTION
**Pull request by beausmith**
Resolves: https://github.com/votingworks/vxmail/issues/150

image

### CODE REVIEW

File: apps/election-manager/src/screens/SmartcardsScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            </Button>
+          </p>
+          {isProgrammingCard && <p>Is programming card…</p>}
+        </Prose>
+      </NavigationScreen>
+      {isProgrammingCard && (
+        <Modal
+          content={
+            <Prose textCenter>
+              <ProgressEllipsis>Programming card</ProgressEllipsis>
```
Comment (carolinemodic) [inline apps/election-manager/src/screens/SmartcardsScreen.tsx:95]: Shouldn't we use the Loading component which wraps this?

**Review by carolinemodic — APPROVED** [review]
Do we expect people to understand what write protection means? That doesn't seem obvious/clear but if we think that will just be explained more in the docs then this looks good.

**Comment by carolinemodic** [comment]
I wonder if there is a grouping where we could include both card-configuration and usb-configuration on this screen ... but I'm not sure what the title for that should be maybe "Config" but that feels too vague... so I suppose the USB stuff can just be its own tab down the line.

File: apps/election-manager/src/screens/SmartcardsScreen.tsx (hunk 1 above)
Comment (beausmith) [reply apps/election-manager/src/screens/SmartcardsScreen.tsx:95]: Yep, you're right. Will do. Not sure why I forgot about it. This thing needs a refactor. Might also be worth considering some other more obvious indeterminate progress indicator.

> Do we expect people to understand what write protection means?

I'd argue that we should get rid of write protection for simplicity, but perhaps there is a use case @benadida can add to clarify. Perhaps if the card being programmed is not blank, we add an additional confirmation step to ensure that they wish to overwrite the data on the card.

**Comment by beausmith** [comment]
> I wonder if there is a grouping where we could include both card-configuration and usb-configuration on this screen ... but I'm not sure what the title for that should be maybe "Config" but that feels too vague... so I suppose the USB stuff can just be its own tab down the line.

That's a thought. Let's brainstorm to see how we might make it work when we get to that task!

**Comment by beausmith** [comment]
@carolinemodic - There are certainly many improvements we can make to this screen. My goal here was to simply port the functionality in this PR.

**Pull request merged**